installer/roles/image_build/tasks/main.yml

```diff
@@ -130,23 +130,9 @@
     mode: '0700'
   delegate_to: localhost
 
-- name: Stage google-cloud-sdk.repo
-  copy:
-    src: google-cloud-sdk.repo
-    dest: "{{ docker_base_path }}/google-cloud-sdk.repo"
-    mode: '0700'
-  delegate_to: localhost
-
-- name: Stage rsyslog.repo
-  copy:
-    src: rsyslog.repo
-    dest: "{{ docker_base_path }}/rsyslog.repo"
-    mode: '0700'
-  delegate_to: localhost
-
 - name: Stage rsyslog.conf
   copy:
-    src: rsyslog.conf
+    src: ../tools/docker-compose/rsyslog.conf
```

Comment:
This references this file???

https://github.com/ansible/awx/blob/7263d1ad949d3accfa2ec79930de519693cab471/tools/docker-compose/rsyslog.conf

assuming I understand... several things about this rub me the wrong way. What is `..` anyway? This file is `installer/roles/image_build/tasks/main.yml`, so how do you get from there to project root? It assumes that it is always ran from the project root? That's not a good practice. It could be rewritten to use the path relative to the role file.

But where should it live anyway? I would rather its home be in `installer/` than `tools/`.

Reply:
Mostly just wanted to get to one file instead of two that could diverge. I'll look at flipping it to have the dev env pull from the image build source file tomorrow.

Reply:
src: "{{ playbook_dir }}/path/to/rsyslog.conf"

Reply:
fixed by @rooftopcellist